flocx_market/api/bid.py

```diff
@@ -2,36 +2,47 @@
 from flask import request
 from flocx_market.objects import bid
 
+import flocx_market.api.app as application
+
 
 class Bid(Resource):
 
     @classmethod
     def get(cls, marketplace_bid_id=None):
+
+        context = application.create_context()
```

Comment:
It may be possible to have create_context automatically called before every request; see the `before_request` decorator and an example here: https://stackoverflow.com/questions/19531721/flask-g-user-and-before-request (there may be better examples).